gcpim.go

```diff
@@ -0,0 +1,121 @@
+package main
+
+import (
+	imtypes "cloud-android-orchestration/api/instancemanager/v1"
+	compute "cloud.google.com/go/compute/apiv1"
+	"context"
+	"errors"
+	"github.com/google/uuid"
+	computepb "google.golang.org/genproto/googleapis/cloud/compute/v1"
+	"google.golang.org/protobuf/proto"
+)
+
+const (
+	// TODO(b/220891296): Make this configurable
+	projectId   = "google.com:cloud-android-jemoreira"
+	sourceImage = "projects/cloud-android-releases/global/images/cuttlefish-google-vsoc-0-9-21"
```

Comment:
These names are internal and should not be available in an open source repo. We're probably going to have to solve design the configuration feature before submitting this PR.

Reply:
Created a minimal config object that is passed to the Instance Manager with at least this two fields removing the values from the code as suggested.